Xamarin.Essentials/FileSystem/FileSystem.ios.tvos.watchos.macos.cs

```diff
@@ -38,6 +38,22 @@ static string GetDirectory(NSSearchPathDirectory directory)
             }
             return dirs[0];
         }
+
+        static string[] PlatformGetAppResourceDirectories(string path)
+        {
+            if (Directory.Exists(path))
```

Comment:
What happens if I change my current directory? Does it always work, or maybe we need to first get the package directory and then append? What if I pass a rooted path, will that still be inside the app resources dir?